lib/disk/modules/VMWareSparseDisk.rb

```diff
@@ -180,4 +299,42 @@ def findFreeSector
     raise "Disk full." if @freeSector * blockSize > @capacity
     @freeSector
   end
+
+  def read_compressed_grain(grain_offset)
+    @vmwareSparseDisk_file.seek(grain_offset, IO::SEEK_SET)
+    buffer = @vmwareSparseDisk_file.read(SIZEOF_SPARSE_EXTENT_COMPRESSED_GRAIN_HEADER)
+    compressed_grain_header = OpenStruct.new(SPARSE_EXTENT_COMPRESSED_GRAIN_HEADER.decode(buffer))
+    buffer = @vmwareSparseDisk_file.read(compressed_grain_header.size)
```

Comment:
Don't think we need the OpenStruct overhead just to pull put a single value

Reply:
Done!